djangocms_alias/filters.py

```diff
@@ -76,3 +80,36 @@ def choices(self, changelist):
                 ),
                 "display": title,
             }
+
+
+if djangocms_versioning_enabled:
+    from djangocms_versioning.constants import UNPUBLISHED
+
+    class UnpublishedFilter(admin.SimpleListFilter):
+        title = _("unpublished")
```

Comment:
```suggestion
        title = _("Unpublished")
```
To be consistent the first character is always upper cased